pkg/daemon/update.go

```diff
@@ -1494,12 +1501,20 @@ func (dn *Daemon) writeUnits(units []ign3types.Unit) error {
 					return err
 				}
 			}
-			// write the unit to disk
 			if err := writeFileAtomicallyWithDefaults(fpath, []byte(*u.Contents)); err != nil {
 				return fmt.Errorf("failed to write systemd unit %q: %v", u.Name, err)
 			}
 
 			glog.V(2).Infof("Successfully wrote systemd unit %q: ", u.Name)
+		} else if u.Mask != nil && !*u.Mask {
+			// if mask is explicitly set to false, make sure to remove a previous mask
+			// see https://bugzilla.redhat.com/show_bug.cgi?id=1966445
+			// note that this does not catch all cleanup cases; for example, if the previous machine config specified Contents, and the current one does not, the previous content will not get cleaned up. For now we're ignoring some of those edge cases rather than introducing more complexity.
```

Comment:
This comment line is a bit long, could you wrap it such that it would be easier to read in an editor? (For example, see below for longer comments)

Reply:
Yes! Was actually wondering, do we have a standard for line length? Could you add that to new hire docs? Or probably even better, could we enable `lll` in `golangci-lint`?

Reply:
> Was actually wondering, do we have a standard for line length? Could you add that to new hire docs?

Good question. I am not sure. Will see if we have any docs on this. I always just went with whatever the standard was for the rest of the file.

> Or probably even better, could we enable lll in golangci-lint?

That might be a good option (although we'd probably have to fix a bunch of comments by hand elsewhere)

Reply:
Just wrapped manually for now